### PR TITLE
Select new rules in RHEL 7CIS Profile

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Update a few selection and references in RHEL7 CIS Profile

### CODE REVIEW

**Comment by yuumasato** [comment]
@openscap-ci test this please

File: rhel7/profiles/cis.profile [hunk 1]
```diff
@@ -684,9 +689,24 @@ selections:
...
+
     ### 6.1.8 Ensure permissions on /etc/group- are configured (Scored)
+    - file_owner_backup_etc_group
+    - file_groupowner_backup_etc_group
+    - file_permissions_backup_etc_group
+
     ### 6.1.9 Ensure permissions on /etc/gshadow- are configured (Scored)
+    - file_owner_backup_etc_gshadow
+    - file_groupowner_backup_etc_gshadow
+    - file_permissions_backup_etc_gshadow
```
Comment (redhatrises) [inline rhel7/profiles/cis.profile]: @yuumasato am I reading this right? 4x the same 3 rules?

Comment (jan-cerny) [reply rhel7/profiles/cis.profile]: No, each time it's a different file: /etc/passwd, /etc/shadow, /etc/group, /etc/gshadow.

Comment (yuumasato) [reply rhel7/profiles/cis.profile]: I'm alleviated there was no copy-pasta error, :smile:

**Pull request merged**